bin/rbenv-each

```diff
@@ -13,21 +13,24 @@ set -e
 [ -n "$RBENV_DEBUG" ] && set -x
 
 # Provide rbenv completions
-if [ "$1" = "--complete" ]; then
-  echo -v
-  exit
-fi
-
-if [ "$1" = "-v" ]; then
-  verbose=1
-  shift
-fi
-
 case "$1" in
-""  | -* )
-  rbenv-help --usage each >&2
-  exit 1
-  ;;
+  --complete )
+    echo --help
+    echo --verbose
+    exit
+    ;;
+  -v | --verbose )
+    verbose=1
+    shift
+    ;;
+  --help )
+    rbenv-help --usage each
```

Comment:
So the overall plan is `--help` prints full help, while minimal --usage is only printed when in error? There is inconsistency in this regard, so once decided in general, we can make the rest eventually consistent.